test/sampler/test_sampler.py

```diff
@@ -211,10 +211,6 @@ def findrng(rng):
 
 # Mark tests that we know are failing on correctedness
 def failing_test(sampler):
-    if isinstance(sampler, nk.sampler.MetropolisSampler):
-        if isinstance(sampler, nkx.sampler.MetropolisPtSampler):
-            return True
-
```

Comment:
Do we need to keep this function?

Reply:
I think we can keep it there in case we need to disable something else in the future